pages/quests/index.tsx

```diff
@@ -87,11 +91,16 @@ const Quests: NextPage = () => {
           </Box>
           <Box p={2}>
             <Box w="100%" p={4}>
-              {keywordSubmitSucceeded ? (
+              {finalTokenUri ? (
```

Comment:
ugh, here it is nesting. we shouldn't nest so deeply at most 2.
so separate it

Reply:
I removed all previous nesting too, as it would have been more complex to keep it.

Plus, we can now have them in UI timeline order on the file.

PTAL at https://github.com/henkaku-center/omise-interface/pull/116/commits/d12b91606d37ffb6ae49b069444707ed5ba38898